content/workshops/rhel_8/exercise2.0.adoc

```diff
@@ -0,0 +1,126 @@
+---
+title: "Exercise 2.0 - Installing Ansible Tower"
+workshops: rhel_8
+workshop_weight: 20
+layout: lab
+---
+
+:license_url: http://ansible-workshop-bos.redhatgov.io/ansible-license.json
+
+:icons: font
+:imagesdir: /workshops/rhel_8/images
+
+
+= Exercise 2.0 - Installing Ansible Tower
+
+== Exercise Description
+
+In this exercise, we are going to get Ansible Tower installed on your control node.
+
+
+=== Step 1: Change directories
+
+Change directories to /tmp
+
+[source,bash]
+----
+cd /tmp
+----
+
+=== Step 2: Download Red Hat Ansible Tower
+
+Download the latest Ansible Tower package
+
+[source,bash]
+----
+curl -O https://releases.ansible.com/ansible-tower/setup/ansible-tower-setup-latest.tar.gz
+----
+
+=== Step 3: Untar and unzip the package file
+
+[source,bash]
+----
+tar xvfz /tmp/ansible-tower-setup-latest.tar.gz
+----
+
+=== Step 4: Change directories
+
+Change directories into the Ansible Tower setup package
+
+[source,bash]
+----
+cd /tmp/ansible-tower-setup-*/
+----
+
+=== Step 5: Open inventory file
+
+Using an editor of your choice, open the inventory file
+
+[source,bash]
+----
+vim inventory
+----
+
+=== Step 6: Identify variables
+
+Fill a few variables out in an inventory file: `admin_password`, `pg_password`, `rabbitmq_password`
+
+[subs=+quotes]
+----
+[tower]
+localhost ansible_connection=local
+
+[database]
+
+[all:vars]
+admin_password=*'ansibleWS'*
+
+pg_host=''
+pg_port=''
+
+pg_database='awx'
+pg_username='awx'
+pg_password=*'ansibleWS'*
+
+rabbitmq_username=tower
+rabbitmq_password=*'ansibleWS'*
+rabbitmq_cookie=cookiemonster
+
+# Isolated Tower nodes automatically generate an RSA key for authentication;
+# To disable this behavior, set this value to false
+# isolated_key_generation=true
+~                                 
+
+----
+
+=== Step 7: Run setup
+
+Run the Ansible Tower setup script
+
+[source,bash]
+----
+sudo ./setup.sh
+----
+
+[NOTE]
+Step 7 will take approx. 10-15 minutes to complete.  This may be a good time to take a break.
+
+
+=== Step 8: Confirm results
+
+At this point, your Ansible Tower installation should be complete.
+You can access your Tower workshop (not forgetting that *workshopname* is the name of your workshop, and *#* is your student number) at:
```

Comment:
Line 112: Replace “not forgetting that” with “Remember:”

Reply:
This section has been removed